ui/hooks/accounts/useMultichainWalletSnapClient.ts

```diff
@@ -62,6 +63,8 @@ export class MultichainWalletSnapClient {
     // However, the balance won't be fetched right away. To workaround this, we trigger the
     // fetch explicitly here (since we are already in a `async` call) and wait for it to be updated!
     await multichainUpdateBalance(account.id);
+    // TODO: Remove this and the above line once Snap account creation flow is async
+    await multichainUpdateTransactions(account.id);
```

Comment:
For future reviewers: We're ok with keeping this logic for now, since this flow is async and prevents the `Multichain*` controllers to properly fetch those info from a Snap (since the Snap account might not be created when the `SnapKeyring:account*Updated` events are fired).

So this is the best approach we have for now.

This will be fixed once the flow is fully async (the fix is on-going).